mobile/src/main.jsx

```diff
@@ -51,12 +51,21 @@ const renderAppWithPersistedState = persistedState => {
     }
   }
 
+  function pingOnceADayWithState () {
+    const state = store.getState()
+    if (state.mobile) {
```

Comment:
I don't know, but I know that if `state.mobile` is `undefined`, then `state.mobile.timestamp` will break the code execution. And I don't want that. 

I can't remember when I saw that, but there is a standard proposal to accept `state.mobile?.timestamp` as a shortcut for `state.mobile ? state.mobile.timestamp : undefined`.